Documentatie/EP-online-validaties.md

```diff
@@ -1,114 +1,98 @@
-|  CheckMainBuildingCertificateByTpg  	|  Wanneer bij woningbouw voor het opnamegebouw (MainBuilding) een adres (TPGIdentification) is opgegeven en er referentiewoningen (ReferenceBuildingList) zijn meegegeven: indien er op de opnamedatum (SurveyDate) al een certificaat is  voor het adres van het opnamegebouw, dan moet het versienummer van dit certificaat gelijk of hoger zijn dan die in het registratiebestand.
-|  CheckMainBuildingUse  	|  Bij utiliteitsbouw moet het primaire gebruik (MainBuildingUse.PrimaryUse) zijn opgegeven. Bij woningbouw mag het primaire gebruik juist niet zijn opgegeven.
-|  CheckMultipleBagBuildingIdsWithMultipleBagResidenceIds	| Bij meerdere Pand-Id’s mogen er niet meerdere VBO-Id’s opgegeven zijn.
-|  CheckNumberOfDwellingsNta  |  Het aantal wooneenheden (NumberOfDwellings) moet bij utiliteitsbouw 0 zijn en bij woningbouw 1 of hoger.
-|  CheckObjectLocationIsSpecified  |  Elk gebouw moet minimaal met één BAG-id (BAGIdentification) of adres (TPGIdentification) worden geïdentificeerd.
-|  CheckSoftwareTool  |  De naam (VendorSoftwareKey) en versienummer (VendorSoftwareVersionId) van de softwaretool moet ingevuld zijn, bestaan als SoftwareTool in EP-online, en daar geldig (actief) zijn op datum van registratie (huidige datum). Het versienummer van het registratiebestand (Version) moet overeenkomen met de XSD versie van de softwaretool in EP-online. De gebruikte rekenmethodiek (TypeCalculation) moet geldig (aangevinkt) zijn bij de softwaretool in EP-online.
-|  CheckSpecificBuildingHasOneBagResidenceId |  Bij een enkel verblijfsobject (Scope = specific) dan moet er ook exact één VBO-Id (BagResidenceId) zijn ingevuld.
-|  CheckStatus  	|  Bij utiliteitsbouw mag de status niet "Woningwaarderingsstelsel" zijn.
-|  CheckSurveyDateHasValue  	|  De opnamedatum (SurveyDate) moet ingevuld zijn.
-|  CheckSurveyDateNotInFuture |  De opnamedatum (SurveyDate) mag niet in de toekomst liggen.
-|  CheckSurveyDateNta |  De opnamedatum (SurveyDate) moet binnen de afmeldperiode voor NTA labels liggen.
-
-## 4. BAG controle
+### 2.4. Conversie van XML naar intern object
+De XML wordt omgezet naar een code-object en alle waarden die van belang zijn voor de verdere verwerking en validatie worden overgezet naar een nieuw ‘monitorfile’ object. Hierna wordt alleen nog maar gebruik gemaakt van dit object.
+Er zitten nog enkele controles, maar deze zouden nooit fout kunnen gaan omdat deze al door de XSD worden afgevangen. Zitten er in voor het geval er een programmeerfout wordt gemaakt, zodat dit tijdens het ontwikkelen nog wordt opgemerkt.
```

Comment:
Deze hele paragraaf is interne informatie en mag in het publieke document weggelaten worden.

graag in de interne versie van het document de tweede alinea van de paragraaf vervangen met "Er worden nog enkele controles uitgevoerd, maar deze zouden nooit fout kunnen gaan aangezien deze al door de XSD validaties worden afgevangen. (Deze validaties zitten er in om programmeerfouten te voorkomen en deze bij het ontwikkelen al te kunnen ontdekken.)"